pager/dlg_pager.c

```diff
@@ -2359,8 +2359,6 @@ int mutt_pager(struct PagerView *pview)
       //  - fp and body->email in special case of viewing an attached email.
       assert(pview->pdata->email); // This should point to the top level email
       assert(pview->pdata->body);
-      assert(pview->pdata->ctx);
-      assert(pview->pdata->ctx->mailbox);
```

Comment:
I'm not super familiar with command-line sending so please feel free to correct my assumptions.

Could this be represented as:
```C
// TUI assertions
if (pview->pdata->ctx)
{
    assert(pview->pdata->ctx->mailbox);
}
```
so the assertion isn't too strict for command-line sending, but ensures a consistent state for TUI usage?

Reply:
I thought I checked and we would never need `pview->pdata->ctx` or `pview->pdata->ctx->mailbox` in this mode, but I will double check!

Reply:
ah, the commit message didn't mention that. if that's the case then it makes sense to drop them.